Gemfile

```diff
@@ -46,7 +46,6 @@ gem "uglifier", ">= 1.3.0"
 gem "waste_exemptions_engine",
     git: "https://github.com/DEFRA/waste-exemptions-engine",
     branch: "main"
```

Comment:
Probably not worth committing an edit to the Gemfile here as it doesn't relate to the task

Reply:
Git rebase done (I think) :)